### DISCUSSION TITLE
After removing cell focus on cell bellow

### DISCUSSION THREAD

**Discussion by RafalSkolasinski** [microsoft/vscode-jupyter#10790] [New Ideas]
In vanilla jupyter lab if you remove cell, it is hit `<ESC>; x`, after removing current cell it focuses on the cell bellow.

VS Code focuses on the cell above which goes against the built muscle memory and makes transition harder.

**Comment by amunger** [comment]
I'm actually seeing it focus on the cell below. What OS are you using?

![focus](https://user-images.githubusercontent.com/2019016/179036643-94fdb573-6848-4810-b4a8-3c14a4c1d490.gif)
